snap/snapcraft.yaml

```diff
@@ -0,0 +1,42 @@
+name: onedrive # you probably want to 'snapcraft register <name>'
```

Comment:
Has this been registered? Is there a conflict?

Reply:
Yes, it was previously registered. But it was never published. We'll have to send a request to claim this name.

Reply:
I tried to fill a name claim, but I was not able to due to [this bug](https://bugs.launchpad.net/snapcraft/+bug/1771767).

Reply:
In the worst case, we can use "onedrive-abraunegg".

Reply:
Ok, I successfully opened a name claim, let's wait to see what they say.

PS: I used the following URL as pointed [here](https://bugs.launchpad.net/snapstore-server/+bug/1771767/comments/7).

https://snapcraft.io/register-name-dispute?snap-name=onedrive

Reply:
Great news, I successfully claimed the `onedrive` name in Snap Store. Feel free to tell me if you happen to want me to transfer the ownership to you.

Reply:
Thats great you have claimed the name, but please - you can retain ownership here for that.